### DISCUSSION TITLE
the various patches for the Burea Church extended sample set - how does one apply the or put them?

### DISCUSSION THREAD

**Discussion by scottmusic** [GrandOrgue/grandorgue#1457] [Help]
I recently had to rebuild my system after a harddrive crash and a motherboard crash, and reinstall everything, including GrandOrgue.

One of the organs I liked is the Burea Church organ - yet there are multiple patches for it.  I can't remember if they were there when I first installed the organ or not.  Does anyone know if one simply does them in order (which is implied but also not clear!), and then copy and paste the contents of each separate patch, successively into the main BureaChurch folder?
(This organ is linked to on the GrandOrgue SourceForge page: https://sourceforge.net/p/ourorgan/samplesets/Burea_church/)

Many thanks,
Scott

**Comment by larspalo** [comment]
@scottmusic Download all the patches, extract them and then apply them by copying the contents into the main sample set folder in the order they appear (not that it should matter too much).

I am currently working on an updated version of the Bureå Church Sample Set in an organ package form, but it will take some time before it's ready. The updated version will have all patches already applied and lots of improvements to the samples (pitch info, corrected loops, corrected cue points and in some cases even a few re-worked samples from the original files I and Graham worked on. There will also be some new graphics and the extended version will have extended compass too.

**Comment by ClarkeBrunt** [comment]
I recently went through installing the original and these patches myself: while @larspalo as the author is clearly the authority on what is in the patches, here's what I found (place the contents of the archives appropriately within the directory of the Burea organs):

BureaChurchUpdated0.3patch.7z - adds an additonal .organ file, burea_church0305.organ, and the associated ConsoleImages, to represent the graphical console (rather than default GO-supplied console). I created an edited version of this .organ file to change the ChurchName field (to e.g. "Bureå Church (GUI)") so that I could distinguish it from the original version in lists displayed by GO.

Patch_BureaChurch_rev1422warnings.7z  - update 3 .organ files (but not the new one supplied in the previous patch) to avoid warnings.

patch1-Burea_Church-beta2.7z  - fix some .wav files

Patch2_BureaChurch.7z  - fix some (other) .wav files

So all the patches are all independent - it doesn't matter in what order you apply them.

As Lars says, if there is eventually a new version including the patches (though maybe with the different ChurchName to distinguish GUI-console from GO-supplied console?), then this will all be irrelevant.

**Reply by larspalo** [reply]
> though maybe with the different ChurchName to distinguish GUI-console from GO-supplied console?

Yes, I intend to alter the ChurchName(s) slightly in the new package so that it will be obvious what version it is.

**Reply by larspalo** [reply]
After having spent considerable time working on this, there is now an updated .orgue package for the Bureå Church sample set. As usual, it's available at http://familjenpalo.se/vpo/download/.

**Reply by jason-barba** [reply]
I have installed it. Its great :-) Thank you, Lars, for your effort!

**Reply by paolo2504** [reply]
> After having spent considerable time working on this, there is now an updated .orgue package for the Bureå Church sample set. As usual, it's available at http://familjenpalo.se/vpo/download/.

Lars, please clarify. Is this new revised version including ALL previous patches, including the 0305 one ? I mean, no need to apply them over this new update ?
Thanks.

**Reply by larspalo** [reply]
> Lars, please clarify. Is this new revised version including all previous patches including the 0305 one ?

Yes, the 0305 is available as additional panels. All previous patches plus much more is included in the .orgue package and there shouldn't be any need of patching. This version has slightly different church names so it can safely co-exist with an older version if someone really wants that.

**Reply by jason-barba** [reply]
As said above I have installed the basic version. But how do I find the extended version? Call me old, stupid or something but I cannot solve this.
//Jens

**Reply by larspalo** [reply]
@jason-barba When the package is registered (dropped into the package directory), there should be two organs available to "Load". Just select the extended version to load. (You might have to adjust the first column a little to see the whole text).

![SampleSetLoadList](https://github.com/GrandOrgue/grandorgue/assets/6173941/be50de09-19f8-4c4c-8a3f-dcef39d04fdf)

**Reply by jason-barba** [reply]
Then something went wrong when installing the package. I only have the basic version in that list.
Re-install on top of the current installation or remove and install again?

**Reply by larspalo** [reply]
If the package is placed in the GO package directory, you should only need to re-start GO to detect the organs. If you've "installed" the package from another directory, I suggest you go into the Settings dialog and remove the installed organ on the Organ tab, then move the .orgue package file to the package directory and restart GO to auto detect it. (It should be possible to install the package again, but why bother with it as it's better to just drop the file in the correct directory to begin with?)

**Reply by jason-barba** [reply]
Will do!
;-)

**Reply by jason-barba** [reply]
I failed :-( 

I removed BureaChurch from Settings/Organs and deleted BureaChurc.orgue from GrandOrgue/OrganPackages. I also found and deleted a BureaChurch.organ i in the GrandOurge/Organs folder. The, downloaded a new BureaChurch.orgue from your site, moved it to the GrandOrgue/OrganPackages and (re)started GO. And there was only the basic version to load; GO even recognized my earlier made MIDI settings. So there must be more files to remove. .idx and/or .cmb? Or other? But the .idx and .cmb file names are not easy to interpret. How do I know which belongs to the BureaChurch sample set?

**Reply by jason-barba** [reply]
So I tried BureaChurch.orgue on my test GO implementation. This has never "seen" BureaChurch.orgue. Once I started GO both version were found under Load. But, when loading the extended version I got an error message (see attached file).
![Screenshot 2023-06-08 191559](https://github.com/GrandOrgue/grandorgue/assets/56149267/a177a49f-4868-462d-b5b9-daa29b841d2b)
I run GO 3.6.7-1 on Ubuntu 22.04 LTS.

**Reply by larspalo** [reply]
Right, it's caused by me using the newer builtin divisional specification. Use a more recent GO version and it should work (it was implemented around 3.9.0-1).

**Reply by jason-barba** [reply]
OK. Will upgrade in due course.
But what about completely removing an organ from GO i.e. how to identify the to the said organ's corresponding .cmb and .idx files?

**Reply by paolo2504** [reply]
Jason, as said by others probably you just need to use the latest release of GO. No problem with that. Additionally, with the new settings options you can easily remove all related organs files.

**Reply by jason-barba** [reply]
Thanks for kind advice!

**Reply by jason-barba** [reply]
I first upgraded to latest GO version 3.12 in my sandbox. No problems, the Extended Bureå Church loaded succesfully.
Upgraded also the actual VPO, same procedure as with the sandbox. GO executes as i should, installed organs working OK with all settings.
But still the Extended Bureå Church doesn't show in the Load menu. The cure re-install BureaChurch.orgue ad modum described in earlier post didn't help, I only see the basic version in the Load menu. Settings/Organs/Delete didn't clear everything.  
What have I missed?

**Reply by jason-barba** [reply]
Problem solved in #1571.
